src/v/cluster/health_monitor_backend.cc

```diff
@@ -271,6 +277,271 @@ void health_monitor_backend::abort_current_refresh() {
     }
 }
 
+namespace {
+void record_risks_in_report(
+  restart_risk_report& report,
+  model::node_id self,
+  const followers_stats& fs,
+  const model::topic_namespace& nt,
+  model::partition_id pid) {
+    // 0 or 1 each
+    uint16_t self_out_of_sync = std::ranges::count(fs.out_of_sync, self);
+    uint16_t self_down = std::ranges::count(fs.down, self);
+    uint16_t self_in_sync = 1 - self_out_of_sync - self_down;
+
+    uint16_t n_in_sync = fs.in_sync;
+    uint16_t n_out_of_sync = fs.out_of_sync.size();
+    uint16_t n_down = fs.down.size();
+    uint16_t n_voters = n_in_sync + n_out_of_sync + n_down;
+
+    partition_risk restart_risk{partition_risk::c::no_risk};
+    vlog(
+      clusterlog.trace,
+      "record_risk_in_report nt={}, pid={}, self_out_of_sync={}, self_down={}, "
+      "self_in_sync={}, n_in_sync={}, n_out_of_sync={}, n_down={}",
```

Comment:
would it be possible to make this log message more readable without having to refer to the code ?

Reply:
It's a trace log message I only left because I used it for debugging. I'm not sure if it is going to be useful, so I'll remove it.